src/DynamoCore/Logging/DynamoLogger.cs

```diff
@@ -242,7 +242,7 @@ public DynamoLogger(DebugSettings debugSettings, string logDirectory, Boolean is
                 cliMode = isCLIMode;
                 serviceMode = isServiceMode;
 
-                if (!testMode && !isServiceMode)
+                if ((!testMode || cliMode) && !isServiceMode)
```

Comment:
Hi @twastvedt Is the previous check `!testMode && !isServiceMode` not true before this change?

Reply:
Not in my testing. I found that testMode was true when I ran the cli. Here, I think: https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoApplications/StartupUtils.cs#L500. The [previous constructor](https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Logging/DynamoLogger.cs#L209-L217), before [this PR](https://github.com/DynamoDS/Dynamo/pull/13860), only checked cliMode, not testMode.